client/src/pages/Faq.js

```diff
@@ -422,7 +422,7 @@ const Faq = () => {
             <p>
               Please check if your password has at least 8 letters. If you
               already have an account but forgot your login password, click on
```

Comment:
Changing your sign in password on line 424

Reply:
updated